src/SixLabors.Fonts/GlyphMetrics.cs

```diff
@@ -201,104 +278,45 @@ internal void RenderDecorationsTo(IGlyphRenderer renderer, Vector2 location, flo
                 Vector2 scale = new Vector2(scaledPPEM) / this.ScaleFactor;
                 Vector2 offset = location + (this.Offset * scale * MirrorScale) + (new Vector2(0, positionY) * scale * MirrorScale);
 
-                Vector2 tl = new(offset.X, offset.Y);
```

Comment:
I've merged the `IGlyphRenderer` implementations since decorations are a first class concern.